packages/cozy-harvest-lib/src/assets/datatypes/icon-contract.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><g fill="none" fill-rule="evenodd"><path fill="#A9CBF9" d="M3.002 43A2.001 2.001 0 0 1 1 41.003V2.997C1 1.894 1.89 1 2.991 1H23l10 10v32H3.002z"/><path fill="#297EF2" d="M23 1l10 10h-8c-1.113 0-2-.895-2-2V1zm3.931 45.452c-1.066.305-1.686-.306-1.38-1.38L27 40.005l13.59-13.592a2 2 0 0 1 2.831.007l2.162 2.162c.784.784.784 2.054.008 2.83L32 45.005l-5.069 1.448zM16 38h10v2H16v-2z"/></g></svg>
```

Comment:
Should we remove the fill attribute?

Reply:
No, those are not monochrome icons, they are more like illustrations that need their colors.